dashboards/flows-by-science-discipline.json

```diff
@@ -2598,9 +2454,10 @@
       {
         "allValue": null,
         "current": {
-          "text": "All",
+          "tags": [],
+          "text": "AMPATH sFlow MLXE",
           "value": [
-            "$__all"
+            "AMPATH sFlow MLXE"
```

Comment:
Was it intended to make the default by this AMPATH sensor? I suspect not but figured I would ask.

Reply:
No, it wasn't - oversight on my part.